app/views/posts/index.html.erb

```diff
@@ -1 +1,31 @@
-<h1>This page shows all posts</h1>
\ No newline at end of file
+<main>
+    <div class="posts-main-container">
+        <div class="single-user">
+            <div class="user-image">
+                <img src="<%= @user.photo %>" alt="profile" class="user-profile">
+            </div>
+            <div class="user-information">
+                <h2 class="user-name"><%= @user.name %></h2>
+                <span class="post-counter"><p class="count">Number of posts: <%= @user.posts.count %></p></span>
+            </div>
+        </div>
+        <div class="user-all-posts">
+            <% @posts.each_with_index do |post, index| %>
+                <div class="user-post">
+                    <h4 class="post">Post #<%= index+1 %></h4>
+                    <div class="post-content">
+                        <p class="post-paragraph">
+                            <%= post.text %>
```

Comment:
- [ ] Please add a link to each post that redirects users to the post show page, which you will render all of the comments for the post.